app/views/players/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for [team, player] do |f| %>
+  <div class="form-group">
+    <%= f.label :name %>
+    <%= f.text_field :name, class: 'form-control', placeholder: "Enter player name" %>
+  </div>
+  <div class="form-group">
+    <%= f.label :age %>
+    <%= f.text_field :age, class: 'form-control', placeholder: "Enter age" %>
```

Comment:
Consider using `number_field` to generate a number input field when the value is numeric: http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-number_field (also see the other field types such as email_field that are available).

By using number_field/email_field where appropriate, touch screen devices will show an optimized keyboard layout when the user focuses on the field. This makes entering numbers/email addresses easier.